### PR TITLE
fix(hitbtc) - remove safeNetwork

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: ts/src/hitbtc.ts [hunk 1]
```diff
@@ -850,7 +850,7 @@ export default class hitbtc extends Exchange {
             for (let j = 0; j < rawNetworks.length; j++) {
                 const rawNetwork = rawNetworks[j];
                 const networkId = this.safeString2 (rawNetwork, 'protocol', 'network');
-                const network = this.safeNetwork (networkId);
+                const network = this.networkIdToCode (networkId);
```
Comment (carlosmiei) [inline ts/src/hitbtc.ts:853]: @ttodua  I'm not sure you can just change the behavior without testing, in this case we were returning every network uppercased and that will not happen with `networkIdToCode`....

Comment (ttodua) [reply ts/src/hitbtc.ts:853]: pushed

File: ts/src/hitbtc.ts [hunk 2]
```diff
@@ -850,7 +850,7 @@ export default class hitbtc extends Exchange {
             for (let j = 0; j < rawNetworks.length; j++) {
                 const rawNetwork = rawNetworks[j];
                 const networkId = this.safeString2 (rawNetwork, 'protocol', 'network');
-                const network = this.safeNetwork (networkId);
+                const network = this.networkIdToCode (networkId).toUpperCase ();
```
Comment (carlosmiei) [inline ts/src/hitbtc.ts:853]: @ttodua what if the network is undefined? toUpperCase might crash

**Comment by ttodua** [comment]
@carlosmiei this passed too ^

**Comment by ttodua** [comment]
@carlosmiei if you can review/merge this PR

**Pull request merged**